website/src/content/docs/components/alert.mdx

```diff
@@ -82,3 +82,40 @@ The `Alert` component has the following props.
     </tr>
   </tbody>
 </table>
+
+## CSS API
+
+import { Styles } from "@gemini-ui-astro/styles";
+
+<Styles />
+
+```astro live
+---
+import "@gemini-ui-css/components/alert.css";
```

Comment:
does the user now have to import a stylesheet for each component they use ?

Reply:
Nah, not if they use the astro components. But if they want to use CSS directly (e.g. if they are not using astro) that is now an option.